### PR TITLE
clarify when to use sqlfluff and sqlfmt

### PR DESCRIPTION
**Pull request by mirnawong1**
clarifying whento use sqlfluff and sqlfmt per [slack thread](https://dbt-labs.slack.com/archives/C02NCQ9483C/p1688396662919929)

### CODE REVIEW

File: website/docs/docs/cloud/dbt-cloud-ide/lint-format.md [hunk 1]
```diff
@@ -183,6 +183,19 @@ To format your Python code, dbt Cloud integrates with [Black](https://black.read
...
 ## FAQs
 
+<details>
+<summary>When to use SQLFluff and when to use sqlfmt?</summary>
+
+SQLFluff and sqlfmt are both tools used for formatting SQL code, but there are some differences that may make one preferable to the other depending on your use case. <br />
+
+SQLFluff is a linter and formatter for SQL code. This means that it doesn't only focus on code formatting but also analyzes your code to identify potential issues and bugs, and follows coding standards. SQLFluff has a set of rules and [customizable configurations](#customize-linting) to ensure consistent coding practices. You can also use SQLFluff to keep your SQL code well-formatted and follows best practices. <br />
+
+sqlfmt is a SQL code formatter. This means it automatically formats your SQL code according to a set of formatting rules which you can't change. It focuses solely on the appearance and layout of the code, which helps ensure consistent indentation, line breaks, and spacing. sqlfmt doesn't analyze your code for errors or bugs and doesn't look at coding issues beyond code formatting. <br />
```
Comment (mirnawong1) [inline website/docs/docs/cloud/dbt-cloud-ide/lint-format.md]:
```suggestion
sqlfmt is a SQL code formatter. This means it automatically formats your SQL code according to a set of formatting rules which isn't customizable. It focuses solely on the appearance and layout of the code, which helps ensure consistent indentation, line breaks, and spacing. sqlfmt doesn't analyze your code for errors or bugs and doesn't look at coding issues beyond code formatting. <br />
```

File: website/docs/docs/cloud/dbt-cloud-ide/lint-format.md [hunk 2]
```diff
@@ -183,6 +183,19 @@ To format your Python code, dbt Cloud integrates with [Black](https://black.read
 
 ## FAQs
 
+<details>
+<summary>When to use SQLFluff and when to use sqlfmt?</summary>
+
+SQLFluff and sqlfmt are both tools used for formatting SQL code, but there are some differences that may make one preferable to the other depending on your use case. <br />
+
+SQLFluff is a linter and formatter for SQL code. This means that it doesn't only focus on code formatting but also analyzes your code to identify potential issues and bugs, and follows coding standards. SQLFluff has a set of rules and [customizable configurations](#customize-linting) to ensure consistent coding practices. You can also use SQLFluff to keep your SQL code well-formatted and follows best practices. <br />
```
Comment (mirnawong1) [inline website/docs/docs/cloud/dbt-cloud-ide/lint-format.md]:
```suggestion
SQLFluff is a SQL code linter and formatter. This means that it doesn't only focus on code formatting but also analyzes your code to identify potential issues and bugs, and follows coding standards. SQLFluff has a set of rules and [customizable configurations](#customize-linting) to ensure consistent coding practices. You can also use SQLFluff to keep your SQL code well-formatted and follows best practices. <br />
```

File: website/docs/docs/cloud/dbt-cloud-ide/lint-format.md [hunk 3]
```diff
@@ -183,6 +183,19 @@ To format your Python code, dbt Cloud integrates with [Black](https://black.read
 
 ## FAQs
 
+<details>
+<summary>When to use SQLFluff and when to use sqlfmt?</summary>
+
+SQLFluff and sqlfmt are both tools used for formatting SQL code, but there are some differences that may make one preferable to the other depending on your use case. <br />
+
+SQLFluff is a SQL code linter and formatter. This means that it doesn't only focus on code formatting but also analyzes your code to identify potential issues and bugs, and follows coding standards. SQLFluff has a set of rules and [customizable configurations](#customize-linting) to ensure consistent coding practices. You can also use SQLFluff to keep your SQL code well-formatted and follows best practices. <br />
```
Comment (mirnawong1) [inline website/docs/docs/cloud/dbt-cloud-ide/lint-format.md]:
```suggestion
SQLFluff is a SQL code linter and formatter. This means that it analyzes your code to identify potential issues and bugs, and follows coding standards. It also formats your code according to a set of rules, which are [customizable](#customize-linting), to ensure consistent coding practices. You can also use SQLFluff to keep your SQL code well-formatted and follow best practices. <br />
```

File: website/docs/docs/cloud/dbt-cloud-ide/lint-format.md [hunk 4]
```diff
@@ -10,7 +10,7 @@ Enhance your development workflow by integrating with popular linters and format
 
 <details>
 <summary>What are linters and formatters? </summary>
-Linters analyze code for errors, bugs, and style issues, while formatters fix style and formatting rules. 
+Linters analyze code for errors, bugs, and style issues, while formatters fix style and formatting rules.  Read more about when to use linters or formatters in the [FAQs](#faqs).
```
Comment (mirnawong1) [inline website/docs/docs/cloud/dbt-cloud-ide/lint-format.md]:
```suggestion
Linters analyze code for errors, bugs, and style issues, while formatters fix style and formatting rules.  Read more about when to use linters or formatters in the <a href="https://docs.getdbt.com/docs/cloud/dbt-cloud-ide/lint-format#faqs">FAQs</a>
```

File: website/docs/docs/cloud/dbt-cloud-ide/lint-format.md [hunk 5]
```diff
@@ -183,6 +183,23 @@ To format your Python code, dbt Cloud integrates with [Black](https://black.read
 
 ## FAQs
 
+<details>
+<summary>When to use SQLFluff and when to use sqlfmt?</summary>
```
Comment (mirnawong1) [inline website/docs/docs/cloud/dbt-cloud-ide/lint-format.md]:
```suggestion
<summary>When should I use SQLFluff and when should I use sqlfmt?</summary>
```

File: website/docs/docs/cloud/dbt-cloud-ide/lint-format.md [hunk 6]
```diff
@@ -183,6 +183,23 @@ To format your Python code, dbt Cloud integrates with [Black](https://black.read
 
 ## FAQs
 
+<details>
+<summary>When should I use SQLFluff and when should I use sqlfmt?</summary>
+
+SQLFluff and sqlfmt are both tools used for formatting SQL code, but there are some differences that may make one preferable to the other depending on your use case. <br />
+
+SQLFluff is a SQL code linter and formatter. This means that it analyzes your code to identify potential issues and bugs, and follows coding standards. It also formats your code according to a set of rules, which are [customizable](#customize-linting), to ensure consistent coding practices. You can also use SQLFluff to keep your SQL code well-formatted and follow best practices. <br />
```
Comment (mirnawong1) [inline website/docs/docs/cloud/dbt-cloud-ide/lint-format.md]:
```suggestion
SQLFluff is a SQL code linter and formatter. This means that it analyzes your code to identify potential issues and bugs, and follows coding standards. It also formats your code according to a set of rules, which are [customizable](#customize-linting), to ensure consistent coding practices. You can also use SQLFluff to keep your SQL code well-formatted and follow styling best practices. <br />
```

File: website/docs/docs/cloud/dbt-cloud-ide/lint-format.md [hunk 7]
```diff
@@ -183,6 +183,23 @@ To format your Python code, dbt Cloud integrates with [Black](https://black.read
...
 ## FAQs
 
+<details>
+<summary>When should I use SQLFluff and when should I use sqlfmt?</summary>
+
+SQLFluff and sqlfmt are both tools used for formatting SQL code, but there are some differences that may make one preferable to the other depending on your use case. <br />
+
+SQLFluff is a SQL code linter and formatter. This means that it analyzes your code to identify potential issues and bugs, and follows coding standards. It also formats your code according to a set of rules, which are [customizable](#customize-linting), to ensure consistent coding practices. You can also use SQLFluff to keep your SQL code well-formatted and follow styling best practices. <br />
+
+sqlfmt is a SQL code formatter. This means it automatically formats your SQL code according to a set of formatting rules which isn't customizable. It focuses solely on the appearance and layout of the code, which helps ensure consistent indentation, line breaks, and spacing. sqlfmt doesn't analyze your code for errors or bugs and doesn't look at coding issues beyond code formatting. <br />
```
Comment (mirnawong1) [inline website/docs/docs/cloud/dbt-cloud-ide/lint-format.md]:
```suggestion
sqlfmt is a SQL code formatter. This means it automatically formats your SQL code according to a set of formatting rules that aren't customizable. It focuses solely on the appearance and layout of the code, which helps ensure consistent indentation, line breaks, and spacing. sqlfmt doesn't analyze your code for errors or bugs and doesn't look at coding issues beyond code formatting. <br />
```

File: website/docs/docs/cloud/dbt-cloud-ide/lint-format.md [hunk 8]
```diff
@@ -183,6 +183,23 @@ To format your Python code, dbt Cloud integrates with [Black](https://black.read
...
+<details>
+<summary>When should I use SQLFluff and when should I use sqlfmt?</summary>
+
+SQLFluff and sqlfmt are both tools used for formatting SQL code, but there are some differences that may make one preferable to the other depending on your use case. <br />
+
+SQLFluff is a SQL code linter and formatter. This means that it analyzes your code to identify potential issues and bugs, and follows coding standards. It also formats your code according to a set of rules, which are [customizable](#customize-linting), to ensure consistent coding practices. You can also use SQLFluff to keep your SQL code well-formatted and follow styling best practices. <br />
+
+sqlfmt is a SQL code formatter. This means it automatically formats your SQL code according to a set of formatting rules that aren't customizable. It focuses solely on the appearance and layout of the code, which helps ensure consistent indentation, line breaks, and spacing. sqlfmt doesn't analyze your code for errors or bugs and doesn't look at coding issues beyond code formatting. <br />
+
+Based on your use case, you can:
```
Comment (mirnawong1) [inline website/docs/docs/cloud/dbt-cloud-ide/lint-format.md]:
```suggestion
You can use either SQLFluff or sqlfmt depending on your preference and what works best for you:
```

File: website/docs/docs/cloud/dbt-cloud-ide/lint-format.md [hunk 9]
```diff
@@ -183,6 +183,23 @@ To format your Python code, dbt Cloud integrates with [Black](https://black.read
...
+
+sqlfmt is a SQL code formatter. This means it automatically formats your SQL code according to a set of formatting rules that aren't customizable. It focuses solely on the appearance and layout of the code, which helps ensure consistent indentation, line breaks, and spacing. sqlfmt doesn't analyze your code for errors or bugs and doesn't look at coding issues beyond code formatting. <br />
+
+You can use either SQLFluff or sqlfmt depending on your preference and what works best for you:
+
+- Use SQLFluff to have your code linted and formatted (meaning analyze fix your code for errors/bugs, and format your styling). It allows you the flexibility to customize your own rules.
+
+- Use sqlfmt to only have your code well-formatted, without analyzing your code for errors and bugs. You can't customize rules with it.
```
Comment (Victoriapm) [inline website/docs/docs/cloud/dbt-cloud-ide/lint-format.md]: Maybe add something about how sqlfmt provides this "out of the box" and that would be the benefit. 

> have your code well-formatted, as the default standard, without the need to customize it

Comment (mirnawong1) [inline website/docs/docs/cloud/dbt-cloud-ide/lint-format.md]:
```suggestion
- Use sqlfmt to only have your code well-formatted without analyzing it for errors and bugs. You can use sqlfmt out of the box, making it convenient to use right away without having to configure it.
```

Comment (mirnawong1) [reply website/docs/docs/cloud/dbt-cloud-ide/lint-format.md]: good shout @Victoriapm ! ok reworded it to the below, wdyt?

>Use sqlfmt to only have your code well-formatted without analyzing it for errors and bugs. You can use sqlfmt out of the box, making it convenient to use right away without having to configure it.

Comment (Victoriapm) [reply website/docs/docs/cloud/dbt-cloud-ide/lint-format.md]: Yes, that's goood!  Go ahead and I'll approve it

**Comment by mirnawong1** [comment]
fab thank you @Victoriapm ! i've enabled auto merge so it should merge whenever you approve it. thanks again for flagging ✨

**Review by Victoriapm — APPROVED** [review]
lgtm 🚀

**Pull request merged**